airbyte-integrations/bases/standard-destination-test/src/main/resources/format_reference.txt

```diff
@@ -0,0 +1,23 @@
+VALID FORMATS:
+
+"2021-1-1"
+"2021-01-01"
+"2021-1-1 01:01:01"
+"2021-1-1 01:01:01 +1"
+"2021-01-01T01:01:01-01:00"
+"2021-1-1T01:01:01 +1:00"
+"2021-01-01 01:01:01"
+"2021-01-01 01:01:01 +0000"
+"2021-01-01T01:01:01Z"
+"2021-01-01T01:01:01-01:00"
+"2021-01-01T01:01:01 +1:00"
+
+
+INVALID FORMATS:
+
+"2021-01-01 01:01:01 UTC"
+"2021-01-01T01:01:01 UTC"
+"2021-01-01T01:01:01 +1"
+"2021-01-01T01:01:01 +0000"
+"2021-01-01T01:01:01+0000"
```

Comment:
I believe this is a valid ISO string no?

Reply:
Correct. Generally speaking all of them are "valid ISO strings", the questions is always what pattern of match do we use to format them or convert. I'll place all of them into 1 list of valid ones.